File: app/src/main/java/com/example/fang/b16traveldomain/routeActivity/RouteContract.java
package com.example.fang.b16traveldomain.routeActivity;

import java.util.List;

public interface RouteContract {

    interface  RouteView{
        void set_route(List<RouteModel> routeModelList);

    }

    interface Route{
        void get_route();
        void set_lattitude_longitude(String start_Lattitude,String  start_Longitude,String  destination_Lattitude, String destination_Longitude);

    }

}

File: app/src/main/java/com/example/fang/b16traveldomain/routeActivity/RouteContract.kt
package com.example.fang.b16traveldomain.routeActivity

interface RouteContract {

    interface RouteView {
        fun set_route(routeModelList: List<RouteModel>)

    }

    interface Route {
        fun get_route()
        fun set_lattitude_longitude(start_Lattitude: String, start_Longitude: String, destination_Lattitude: String, destination_Longitude: String)

    }

}
